src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/umb-block-list-property-editor.html

```diff
@@ -18,47 +18,10 @@
                     <div class="__plus" ng-style="{'left':inlineCreateButtonCtrl.plusPosX}">+</div>
                 </button>
 
-                <div class="umb-block-list__block" ng-class="{'--active':layout.$block.active}">
-
-                    <umb-block-list-scoped-block
-                        ng-if="layout.$block.config.stylesheet"
-                        class="umb-block-list__block--content blockelement__draggable-element"
```

Comment:
@Shazwazza this class "blockelement__draggable-element" did slip our review, I´m adding it in a new PR.